mvn-extension/src/main/java/org/arquillian/smart/testing/mvn/ext/SmartTestingMavenConfigurer.java

```diff
@@ -42,16 +42,17 @@
     @Requirement
     public org.codehaus.plexus.logging.Logger mavenLogger;
 
-    private final ChangeStorage changeStorage = new LocalChangeStorage(".");
+    private final ChangeStorage changeStorage = new LocalChangeStorage();
 
     private Configuration configuration;
 
     private Boolean skipExtensionInstallation;
 
     @Override
     public void afterProjectsRead(MavenSession session) throws MavenExecutionException {
+        File projectDirectory = session.getCurrentProject().getModel().getProjectDirectory();
         Log.setLoggerFactory(new MavenExtensionLoggerFactory(mavenLogger));
-        configuration = Configuration.load();
+        configuration = Configuration.load(projectDirectory);
```

Comment:
Suppose this is multi-module project:
```xml
parent
 - config
   -- api
     --- pom.xml
   -- impl-base
      --- pom.xml
   -- spi
      --- pom.xml
 - smart-testing.yml
```
You are trying to execute tests using following:
```bash
cd parent && mvn clean test
```
Assume you have configuration file present in parent dir. Then this method will take wrong configuration as it couldn't find configuration in it's respective module i.e. config.

Here we should look give correct projectdir to `Configuration.load(), otherwise it won't work as per expectation.

Reply:
I probably don't understand this comment.
is the command correct? `cd parent && mvn clean test` ?
if I execute it from parent and the file is in parent, then it should be fine, right?
If you execute it from some submodule, then it won't find the file, which is fine as well I guess. Otherwise, you would need to go through all parent directories looking for some occurrence. I'm not sure if this is correct. If you execute maven build from subdirectory that doesn't contain "pom" file, then it will fail and won't try to find any pom file in the parent directories.

Reply:
but in our case assume that in future we support configuration file per module in future then it will work as per expected if user is executing test goal in sub-module containing config-file.

Assume user is trying to run test from submodule directory. 
sub-module doesn't contain any `smart-testing.yml` but root of project contains `smart-testing.yml` where all smart-testing configuration present.

So Ideally in our case it should take configuration from parent `smart-testing.yml`.

Now it's looking for `smart-testing.yml` in submodule where is it not present. then it'll take all default configuration which is wrong